tests/integration/tests.mk

```diff
@@ -194,3 +194,17 @@ test.integration.race.native: | $(JUNIT_REPORT)
 	$(GO) test -race -p 1 ${T} ${TEST_PACKAGES} -timeout 120m \
 	--istio.test.env native \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
+
+# Defines a target to run a minimal reachability testing basic traffic
+.PHONY: test.integration.kube.reachability
+test.integration.kube.reachability: istioctl | $(JUNIT_REPORT)
+	PATH=${PATH}:${ISTIO_OUT} $(GO) test -p 1 ${T} ./tests/integration/security/ ${_INTEGRATION_TEST_WORK_DIR_FLAG} ${_INTEGRATION_TEST_CIMODE_FLAG} -timeout 30m \
```

Comment:
I'm just starting to look into running tests inside the build container on a Mac, but I suspect putting ISTIO_OUT on the Path won't work in this case. Today ISTIO_OUT points to Mac executables, and we are running inside a linux container. It might be as simple as using LOCAL_OUT which should be pointing at the linux executables.

Reply:
This is copying all the other targets. I agree we should fix it, but this is fixing a 1.5 regression so I think we need to get this in before we look into it more if that works for you?